
{'date': 'Sun Jul 22 2018 20:50:22.982 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 20:55:32.776 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 158
    Tabuleiro ()
  module <module> line 27
    self.linhaA1 = Elemento(QBRANCO, style = dict(
NameError: name 'QBRANCO' is not defined
'''},
{'date': 'Sun Jul 22 2018 21:02:33.383 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 162
    Tabuleiro ()
  module <module> line 26
    self.fase1 = Cena(img = FASE1, style = dict(
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Sun Jul 22 2018 21:02:34.588 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 162
    Tabuleiro ()
  module <module> line 26
    self.fase1 = Cena(img = FASE1, style = dict(
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},